CUE4Parse/UE4/Oodle/Objects/FOodleDictionaryArchive.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using CUE4Parse.UE4.Readers;
+using Newtonsoft.Json;
+
+namespace CUE4Parse.UE4.Oodle.Objects
+{
+    public class FOodleDictionaryArchive
+    {
+        private readonly int MAX_COMPRESS_BUFFER = (1024 * 1024 * 2047);
+        private readonly int OODLE_DICTIONARY_SLACK = 65536;
+        private FArchive InnerArchive;
+        public readonly FDictionaryHeader Header;
+        public readonly byte[] DictionaryData;
+        public readonly byte[] CompactCompresorState;
+
+        public FOodleDictionaryArchive(FArchive Ar)
+        {
+            InnerArchive = Ar;
+            Header = new FDictionaryHeader(Ar);
+            SerializeOodleDecompressData(Header.DictionaryData, out DictionaryData, out UInt32 _);
```

Comment:
Any usages of UInt32 should be changed to the type keyword "uint"

Reply:
This specific usage can be changed to `out _` as the out isn't used.

Reply:
can be removed anyway, it's the length of DictionaryData